Library/src/main/java/Book.java

```diff
@@ -0,0 +1,12 @@
+public class Book {
+    String name;
+    String author;
+    String genre;
+    String descrip;
+    public Book(String nameOfBook, String authorOfBook, String genreOfBook, String description ) {

```

Comment:
Skoro jesteśmy w klasie Book, to nie musimy zmiennych nazywać xxxOfBook.
Możesz użyć tej konstrukcji:
this.author = author;

Ale to ogólnie jest pierdoła, po prostu przyjemny standard

Reply:
Ewentualnie, jak sobie zrobisz klasę z polami, to możesz w intelliJ użyć skrótu do wygenerowania konstruktora. Alt + Insert